data_explorer/app/pages/dataset.py

```diff
@@ -1,15 +1,41 @@
 """Data exploration page of the app."""
+import typing as t
 
 import streamlit as st
 from df_helpers.fields import get_image_fields
 from df_helpers.image_render import configure_image_builder, convert_image_column
 from interfaces.dataset_interface import DatasetLoaderApp
-from st_aggrid import AgGrid, ColumnsAutoSizeMode, GridOptionsBuilder
+from st_aggrid import AgGrid, ColumnsAutoSizeMode, GridOptionsBuilder, JsCode
 
 
 class DatasetExplorerApp(DatasetLoaderApp):
     @staticmethod
-    def setup_app_page(dataframe, fields):
+    def _get_hover_vis_script(field: str) -> JsCode:
```

Comment:
Is there a reason to add this custom javascript? I think the original style fits better with the style of the data explorer.

Reply:
Removed it, I thought having some contrast might be a bit nicer but we can keep it simple